### PR TITLE
Big update to databae seeder

### PR DESCRIPTION
**Pull request by matt-wigg**
1) It now imports images randomly throughout all reviews
2) It now assigns 30 additional reviews to id 0 (Moon id)
3) It now assigns 5 moon images to id 0 (Moon id)

4) Breaks down Promise compiling into a helper function

@miraclestyle @sidpochampally @andrewdeanyoder - Thank you

### CODE REVIEW

File: server/database/dbSeed.js [hunk 1]
```diff
@@ -35,29 +35,73 @@ const databaseInsertion = (data) => new Promise((resolve, reject) => {
...
+  return promiseArray;
+};
+
+const databaseSeeder = () => {
+  const mainReviewCounter = Math.floor(Math.random() * 25) + 10;
+  return Promise.all(promiseCompiler(mainReviewCounter, reviewGenerator))
+    .then(() => console.log('🚀🚀 review database seeded!'))
+    .then(() => {
+      const imageCounter = Math.floor(Math.random() * 300) + 100;
+      return Promise.all(promiseCompiler(imageCounter, databaseImageInsertion))
```
Comment (miraclestyle) [inline server/database/dbSeed.js:84]: @Matt-Wigg , you are calling the promiseCompiler with two arguments. I see that it takes four. Is this working?

Comment (miraclestyle) [reply server/database/dbSeed.js:84]: I can see that the parameters in the referenced functions are optional. This should be ok then.

File: server/database/dbSeed.js [hunk 2]
```diff
@@ -35,29 +35,73 @@ const databaseInsertion = (data) => new Promise((resolve, reject) => {
...
+        .then(() => console.log('🌜🌜 moon reviews seeded!'))
+        .catch((err) => console.error(err));
+    })
+    .then(() => {
+      let moonImageCount = 5;
+      const moonImagesPromises = [];
+      while (moonImageCount > 0) {
+        moonImagesPromises.push(databaseImageInsertion('0', moonImageCount));
+        moonImageCount -= 1;
+      }
```
Comment (miraclestyle) [inline server/database/dbSeed.js]: @Matt-Wigg , this looks like the code in the promiseCompiler. Do you think you can DRY it up?

**Review by miraclestyle — APPROVED** [review]
Everything seems to be in order at the moment.

File: server/database/dbSeed.js (hunk 2 above)
Comment (matt-wigg) [reply server/database/dbSeed.js]: I tried, but I need to pass down an everchanging image count so I insert images 1.jpg through 5.jpg ☹️

**Pull request merged**